Makefile

```diff
@@ -65,9 +65,8 @@ test: $(SOURCE) VERSION .gobuild
 	    $(TEST_COMMAND)
 
 lint:
-	for source_file in $(SOURCE); do \
-		go vet -x $$source_file ; \
-	done
```

Comment:
more tools that need to be installed locally. Are they not part of the go docker image?